app/api/groups/get_granted_permissions.go

```diff
@@ -170,7 +170,7 @@ func (srv *Service) getGrantedPermissions(w http.ResponseWriter, r *http.Request
 
 		sourceGroupsQuery = store.Groups().
 			Where("groups.type != 'User'").
-			Where("id IN ?", managedGroupsQuery.SubQuery()).
+			Where("id IN (?)", managedGroupsWithCanGrantGroupAccessIds).
```

Comment:
Mmh ok, that's definitely something we want to get rid of as soon as mysql is fixed. `id IN (list of ids)` may have very bad perf if there are many of ids (unlike the subquery version).

Reply:
fixed. issue created